indra/newview/llviewertexture.cpp

```diff
@@ -96,6 +96,7 @@ const S32 MAX_CACHED_RAW_IMAGE_AREA = 64 * 64;
 const S32 MAX_CACHED_RAW_SCULPT_IMAGE_AREA = LLViewerTexture::sMaxSculptRez * LLViewerTexture::sMaxSculptRez;
 const S32 MAX_CACHED_RAW_TERRAIN_IMAGE_AREA = 128 * 128;
 const S32 DEFAULT_ICON_DIMENTIONS = 32;
+const S32 DEFAULT_THUMBNAIL_DIMENTIONS = 256;
```

Comment:
DEFAULT_THUMBNAIL_DIMENSIONS

Reply:
Thank you, will fix

Reply:
Please also fix the name of the constant right above: DEFAULT_ICON_DIMENSIONS